docs/monitor/logging.md

```diff
@@ -18,7 +18,7 @@
 
 # Monitoring Kyuubi - Logging System
 
-Kyuubi uses [Apache Log4j](https://logging.apache.org/) for logging.
```

Comment:
Kyuubi uses [Apache Log4j2](https://logging.apache.org/log4j/2.x/) for logging since v1.6.0, for ......

Reply:
hmm... do I have to give you the whole sentence to copy?

Kyuubi uses [Apache Log4j2](https://logging.apache.org/) for logging since version xxx. For versions xxx' and below, it uses [Apache Log4j](https://logging.apache.org/).

Reply:
XD